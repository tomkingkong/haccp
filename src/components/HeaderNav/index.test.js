import React from 'react';
import { shallow } from 'enzyme';

import { HeaderNav } from '.';

describe('HeaderNav', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<HeaderNav />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

