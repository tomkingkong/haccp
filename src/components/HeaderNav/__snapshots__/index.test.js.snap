// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderNav should match snapshot 1`] = `
<nav
  className="header-nav"
>
  <h1
    className="header-text"
    onClick={[Function]}
  >
    HACCP Planner
  </h1>
  <div
    className="nav-btns"
  >
    <button
      className="signup-login logout"
      onClick={[Function]}
    >
      Log Out
    </button>
  </div>
</nav>
`;

exports[`HeaderNav should match snapshot with path / 1`] = `
<nav
  className="header-nav"
>
  <h1
    className="header-text"
    onClick={[Function]}
  >
    HACCP Planner
  </h1>
  <div
    className="nav-btns"
  >
    <button
      className="signup-login log"
      onClick={[Function]}
    >
      Log In
    </button>
    <button
      className="signup-login sign"
      onClick={[Function]}
    >
      Sign Up
    </button>
  </div>
</nav>
`;
